server/public/channels.go

```diff
@@ -0,0 +1,7 @@
+package public
+
+type CallsChannel struct {
+	ChannelID string         `json:"channel_id"`
+	Enabled   bool           `json:"enabled"`
+	Props     map[string]any `json:"props,omitempty"`
```

Comment:
Porting this comment from Figma to add some context:

>  I am considering adding a Props column here as it will let us extend it with custom stuff without having to alter the schema. For example, one interesting use case would be to have certain call configurations specific to the channel (e.g. start recording automatically).